src/utils/VerifiableCredentialService.ts

```diff
@@ -33,22 +34,20 @@ export class VerifiableCredentialService {
 
     async generateSignedVerifiableCredentialJwt(sessionItem: ISessionItem | undefined, getNow: () => number): Promise<string> {
     	const now = getNow();
-    	const subject = sessionItem?.clientId as string;
-    	const givenNames = sessionItem?.given_names?.join(" ");
-    	const famiyNames = sessionItem?.family_names?.join(" ");
-    	const verifiedCredential: VerifiedCredential = new VerifiableCredentialBuilder(givenNames, famiyNames, sessionItem?.date_of_birth, sessionItem?.document_selected, sessionItem?.date_of_expiry)
+    	const subject = sessionItem?.subject as string;
+    	const nameParts = this.buildVcNamePart(sessionItem?.given_names, sessionItem?.family_names);
+    	const verifiedCredential: VerifiedCredential = new VerifiableCredentialBuilder(nameParts, sessionItem?.date_of_birth, sessionItem?.document_selected, sessionItem?.date_of_expiry)
```

Comment:
we don't need `document_selected` and `date_of_expiry` in the VC do we?

Reply:
Yes, thanks!